handlers/misttriggers/stream_buffer.go

```diff
@@ -50,6 +51,7 @@ func (d *MistCallbackHandlersCollection) TriggerStreamBuffer(w http.ResponseWrit
 	}
 
 	rawBody, _ := json.Marshal(body)
+	go d.broker.TriggerStreamBuffer(ctx, body)
```

Comment:
If you run this in an async manner, I wonder what is the value of waiting in the broker itself?

I think that if you need an async workflow here, then probably there is no use in `group.Wait()` in the broker's trigger implementation.